### ISSUE TITLE
TypeError: can't compare offset-naive and offset-aware datetimes

### ISSUE THREAD

**Issue by andyreagan** [fivetran/great_expectations#1509]
**Describe the bug**
I'm writing my first checkpoint, but it won't run:
```
Error running action with name update_data_docs
Traceback (most recent call last):
  File "/Library/Frameworks/Python.framework/Versions/3.8/lib/python3.8/site-packages/great_expectations/validation_operators/validation_operators.py", line 288, in _run_actions
    action_result = self.actions[action["name"]].run(
  File "/Library/Frameworks/Python.framework/Versions/3.8/lib/python3.8/site-packages/great_expectations/validation_operators/actions.py", line 40, in run
    return self._run(
  File "/Library/Frameworks/Python.framework/Versions/3.8/lib/python3.8/site-packages/great_expectations/validation_operators/actions.py", line 332, in _run
    self.data_context.build_data_docs(
  File "/Library/Frameworks/Python.framework/Versions/3.8/lib/python3.8/site-packages/great_expectations/core/usage_statistics/usage_statistics.py", line 242, in usage_statistics_wrapped_method
    result = func(*args, **kwargs)
  File "/Library/Frameworks/Python.framework/Versions/3.8/lib/python3.8/site-packages/great_expectations/data_context/data_context.py", line 1436, in build_data_docs
    index_page_resource_identifier_tuple = site_builder.build(
  File "/Library/Frameworks/Python.framework/Versions/3.8/lib/python3.8/site-packages/great_expectations/render/renderer/site_builder.py", line 265, in build
    index_page_resource_identifier_tuple = self.site_index_builder.build()
  File "/Library/Frameworks/Python.framework/Versions/3.8/lib/python3.8/site-packages/great_expectations/render/renderer/site_builder.py", line 686, in build
    validation_result_keys = sorted(
TypeError: can't compare offset-naive and offset-aware datetimes
Traceback (most recent call last):
  File "/Library/Frameworks/Python.framework/Versions/3.8/bin//great_expectations", line 8, in <module>
    sys.exit(main())
  File "/Library/Frameworks/Python.framework/Versions/3.8/lib/python3.8/site-packages/great_expectations/cli/cli.py", line 65, in main
    cli()
  File "/Library/Frameworks/Python.framework/Versions/3.8/lib/python3.8/site-packages/click/core.py", line 764, in __call__
    return self.main(*args, **kwargs)
  File "/Library/Frameworks/Python.framework/Versions/3.8/lib/python3.8/site-packages/click/core.py", line 717, in main
    rv = self.invoke(ctx)
  File "/Library/Frameworks/Python.framework/Versions/3.8/lib/python3.8/site-packages/click/core.py", line 1137, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "/Library/Frameworks/Python.framework/Versions/3.8/lib/python3.8/site-packages/click/core.py", line 1137, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "/Library/Frameworks/Python.framework/Versions/3.8/lib/python3.8/site-packages/click/core.py", line 956, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/Library/Frameworks/Python.framework/Versions/3.8/lib/python3.8/site-packages/click/core.py", line 555, in invoke
    return callback(*args, **kwargs)
  File "/Library/Frameworks/Python.framework/Versions/3.8/lib/python3.8/site-packages/great_expectations/cli/mark.py", line 28, in wrapper
    func(*args, **kwargs)
  File "/Library/Frameworks/Python.framework/Versions/3.8/lib/python3.8/site-packages/great_expectations/cli/checkpoint.py", line 188, in checkpoint_run
    results = context.run_validation_operator(
  File "/Library/Frameworks/Python.framework/Versions/3.8/lib/python3.8/site-packages/great_expectations/core/usage_statistics/usage_statistics.py", line 242, in usage_statistics_wrapped_method
    result = func(*args, **kwargs)
  File "/Library/Frameworks/Python.framework/Versions/3.8/lib/python3.8/site-packages/great_expectations/data_context/data_context.py", line 871, in run_validation_operator
    return validation_operator.run(
  File "/Library/Frameworks/Python.framework/Versions/3.8/lib/python3.8/site-packages/great_expectations/validation_operators/validation_operators.py", line 238, in run
    batch_actions_results = self._run_actions(
  File "/Library/Frameworks/Python.framework/Versions/3.8/lib/python3.8/site-packages/great_expectations/validation_operators/validation_operators.py", line 301, in _run_actions
    raise e
  File "/Library/Frameworks/Python.framework/Versions/3.8/lib/python3.8/site-packages/great_expectations/validation_operators/validation_operators.py", line 288, in _run_actions
    action_result = self.actions[action["name"]].run(
  File "/Library/Frameworks/Python.framework/Versions/3.8/lib/python3.8/site-packages/great_expectations/validation_operators/actions.py", line 40, in run
    return self._run(
  File "/Library/Frameworks/Python.framework/Versions/3.8/lib/python3.8/site-packages/great_expectations/validation_operators/actions.py", line 332, in _run
    self.data_context.build_data_docs(
  File "/Library/Frameworks/Python.framework/Versions/3.8/lib/python3.8/site-packages/great_expectations/core/usage_statistics/usage_statistics.py", line 242, in usage_statistics_wrapped_method
    result = func(*args, **kwargs)
  File "/Library/Frameworks/Python.framework/Versions/3.8/lib/python3.8/site-packages/great_expectations/data_context/data_context.py", line 1436, in build_data_docs
    index_page_resource_identifier_tuple = site_builder.build(
  File "/Library/Frameworks/Python.framework/Versions/3.8/lib/python3.8/site-packages/great_expectations/render/renderer/site_builder.py", line 265, in build
    index_page_resource_identifier_tuple = self.site_index_builder.build()
  File "/Library/Frameworks/Python.framework/Versions/3.8/lib/python3.8/site-packages/great_expectations/render/renderer/site_builder.py", line 686, in build
    validation_result_keys = sorted(
TypeError: can't compare offset-naive and offset-aware datetimes
```

**To Reproduce**
...

**Environment (please complete the following information):**
 - OS: OSX
 - GE Version: 0.11.0

**Comment by andyreagan** [comment]
I'm going to try running the example in the tutorial on my machine, to see if it's just my machine...

And it's not just my machine being misconfigured. The tutorial worked just fine.

**Comment by andyreagan** [comment]
I wasn't able to reproduce this on the sample dataset pushed to a sample database (it works there, but still not working in production). Without a reproducible example I'm not sure what help anyone here can offer, feel free to close or we can let it go stale.

The one difference that I am aware of between production server and MWE I just built is that the former is connecting over SSL.

**Comment by andyreagan** [comment]
I think I found, maybe, the culprit. If I look in the folder of the validations, I see:
```
    └── validations
        ├── schema
        │   └── table
        │       └── warning
        │           ├── 20200529T105432.202406Z
        │           │   └── 2020-05-29T10:54:32.205009+00:00
        │           │       └── f25b7348659c994981eabc758bbd4a74.json
        │           ├── 20200529T110436.902492Z
        │           │   └── 2020-05-29T11:04:36.902580+00:00
        │           │       └── 035612aeb81457b5236b70d05d0e4022.json
        │           ├── 20200529T110723.801671Z
        │           │   └── 2020-05-29T11:07:23.801744+00:00
        │           │       └── 035612aeb81457b5236b70d05d0e4022.json
        │           ├── 20200529T111029.884458Z
        │           │   └── 2020-05-29T11:10:29.884553+00:00
        │           │       └── 035612aeb81457b5236b70d05d0e4022.json
        │           ├── 20200529T111227.828719Z
        │           │   └── 2020-05-29T11:12:27.828807+00:00
        │           │       └── 035612aeb81457b5236b70d05d0e4022.json
        │           ├── 20200529T143541.153837Z
        │           │   └── 2020-05-29T14:35:41.153933+00:00
        │           │       └── 035612aeb81457b5236b70d05d0e4022.json
        │           ├── 20200529T143810.706999Z
        │           │   └── 2020-05-29T14:38:10.707091+00:00
        │           │       └── f708d4c56758441ae7bdfdeab93c5852.json
        │           └── expanding_expectations
        │               ├── 2020-05-29T06:51:03.330882
        │               │   └── f25b7348659c994981eabc758bbd4a74.json
        │               ├── 2020-05-29T06:51:08.485865
        │               │   └── f25b7348659c994981eabc758bbd4a74.json
        │               ├── 2020-05-29T06:52:03.232765
        │               │   └── f25b7348659c994981eabc758bbd4a74.json
        │               ├── 2020-05-29T06:53:29.484617+00:00
        │               │   └── f25b7348659c994981eabc758bbd4a74.json
        │               ├── 2020-05-29T10:50:49.004218
        │               │   └── f25b7348659c994981eabc758bbd4a74.json
        │               └── 2020-05-29T10:54:17.854036+00:00
        │                   └── f25b7348659c994981eabc758bbd4a74.json
```
some of these have a tz stamp, some don't...

**Comment by andyreagan** [comment]
I wiped folder inside of the data docs, and now it's working...

Not sure how this happened under the hood, but maybe this will offer some info a future person.

Going to leave it open as a bug, again, feel free to close (without an example of how to get to this state).

**Comment by andyreagan** [comment]
I managed to reproduce it again in the course of working with GE, and now I see how.

When you edit an expectation with the `edit` command in a jupyter notebook, the last cell uses ```datetime.utcnow()``` as the date. This is timezone unaware, and breaks things (since you'll already have validations from the CLI that were written with timezone-aware stamps.

The fix is to replace this call (first `import pytz`), with:
```
datetime.utcnow().replace(tzinfo=pytz.UTC)
```

**Comment by Aylr** [comment]
@andyreagan Thanks for this super detailed report! As it turns out we have been working on this issue and shipped a release late last night with a fix so you'll want to grab `0.11.1` https://github.com/great-expectations/great_expectations/releases/tag/0.11.1

**Comment by andyreagan** [comment]
Awesome, thanks. I see it right at the top of the release notes.

I was thinking a better idea might be to not save the tz in the folder name itself, I'll have to look at the code to see what y'all did.
